flink-ml-lib/src/main/java/org/apache/flink/ml/operator/batch/classification/LogisticRegressionTrainBatchOp.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.operator.batch.classification;
+
+import org.apache.flink.ml.operator.batch.linear.BaseLinearModelTrainBatchOp;
+import org.apache.flink.ml.operator.common.LinearModelType;
+import org.apache.flink.ml.param.Param;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * {@link org.apache.flink.ml.operator.batch.BatchOperator} implementation fot logistic regression
+ * training.
+ */
+public class LogisticRegressionTrainBatchOp
```

Comment:
Is LogisticRegressionTrainBatchOp expected to do the training? This class is not used by any Estimator subclasses. Should it implement the Estimator interface?